src/theme/palette.ts

```diff
@@ -6,24 +6,114 @@
 import { alpha, PaletteOptions } from '@mui/material';
 
 // Palette and PalleteOptions types can be extended here as needed
-/*
+
 declare module '@mui/material/styles/createPalette' {
   interface Palette {
```

Comment:
no longer needed?

Reply:
I think it's still needed to extend `buy` and `sell` these two types inside palette. Based on Troy's previous comment, I changed `buy` and `sell`'s belong type.

Reply:
yes interface `PaletteOptions` is needed, but you're not using `Palette` anywhere right?

Reply:
I got the same question with you at first. But if we need to use extended theme in other component, we also need to define in the `Palette`, if I didn't misunderstand this article: [https://javascript.plainenglish.io/extend-material-ui-theme-in-typescript-a462e207131f](url)

Reply:
awesome!